README.md

```diff
@@ -30,10 +30,7 @@
   "githubUrl": "<GITHUB REPOSITORY URL>",
   "npmPkg": "<OPTIONAL NPM PACKAGE NAME>",
   "nameOverride": "<OPTIONAL PACKAGE DISPLAY NAME>",
-  "examples": [
-    "<THE URL TO REPO>",
-    "<THE URL TO A SNACK>"
-  ],
+  "examples": ["<THE URL TO REPO>", "<THE URL TO A SNACK>"],
```

Comment:
This format was intended, since we want multiple entries to be listed in separate lines in data field. This pattern also reflects out auto-format rules.

Reply:
Reverted this example formatting change.